src/main/resources/static/js/login.js

```diff
@@ -0,0 +1,20 @@
+let formElem = document.querySelector('#form');
+
+let buttonElem = formElem.button1;
+
+buttonElem.addEventListener('click', function () {
+let idValue = formElem.mid.value
+let pwValue = formElem.mpw.value;
+    // console.log(idValue);
+    // console.log(pwValue);
+    fetch('/login',{
+        method: 'post',
+        headers: {
+            'Content-Type': 'application/json'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/koreait0000/MNDC/pullRequest?prid=7786920)
```suggestion
            "Content-Type": 'application/json'
```